docs/community.aws.aws_batch_job_definition_module.rst

```diff
@@ -28,8 +28,8 @@ Requirements
 The below requirements are needed on the host that executes this module.
 
 - python >= 3.6
-- boto3 >= 1.15.0
-- botocore >= 1.18.0
+- boto3 >= 1.17.0
+- botocore >= 1.20.0
```

Comment:
This must not change/increase in the stable-2 release.  
I guess you must removeyour local amazon.aws >= 3.0.0 collection and install the latest stable-2 release before generating the docs.

Reply:
understood, pushed the updated code.
I switched to [amazon.aws/stable-2](https://github.com/ansible-collections/amazon.aws/tree/stable-2) locally, then ran the `add_docs`, does that sound correct?